withQuery.js

```diff
@@ -14,19 +15,26 @@ export default function (handler, _config = {}) {
         const queryContainer = withQueryContainer(component);
 
         if (!config.reactive) {
-            const staticQueryContainer = withStaticQuery(queryContainer);
+            const StaticQueryContainer = withStaticQuery(queryContainer);
 
             return function (props) {
                 const query = handler(props);
 
-                return React.createElement(staticQueryContainer, {
-                    query,
-                    props,
-                    config
-                })
+                return (
+                    <SSRDataStoreContext.Consumer>
```

Comment:
I fail to understand why you need context, if you can use `window` as a global variable to do your thing. 
```js
window.GRAPHER = { queryStore: {}, hydrationCompleted: bool, dataStore }
```